WebRoot/js/zimbraMail/briefcase/model/ZmBriefcaseItem.js

```diff
@@ -359,9 +359,16 @@ function() {
 
 ZmBriefcaseBaseItem.prototype._loadFromDom =
 function(node) {
-
-	this.id = node.id;
-
+    /* In search api response we get 'id' and 'sfid' for files in 'files shared with me' folder. Here id is a combination id which is required to form the rest url and get the prview data and sfid is the actual 'id' of the item/file.
+    To delete the file need to call "ItemActionRequest' with 'sfid' coming from 'searchResponse' since this is the actual file id.
+    Interchanged the 'id' and 'sfid' properties for items in 'files shared with me' folder, so that rest of the operations is perfomed as earlier.
+	*/
+	if(node.sfid && node.l == ZmFolder.ID_FILE_SHARED_WITH_ME)  {
+	    this.id = ""+ node.sfid;
```

Comment:
why do we need to do conversion in string, in which other format this data can arrive from server?

Reply:
the sfid comes in "number" format

Reply:
soap api gives everything in string only, are we doing conversion to number format in our code?

Reply:
Reverted these changes as it is not required.